taqueria-vscode-extension/src/lib/gui/ObservableConfig.ts

```diff
@@ -1,3 +1,4 @@
+import { execSync } from 'child_process';
```

Comment:
This import doesn't look like it's used in this file

Reply:
Yeah, good point. I removed it.